[skin.amber] v3.2.23

bartolomesoriano opened the pull request:
### Description
Amber version 3.2.23
### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-skins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

ronie commented:
thx for the update!

could you please remove the BOM character at the start of this file:
script-embuary-image.xml



there seem to be some strings missing in your language file:

Label not defined: 31497
Includes_Shelf_Contents.xml: line 30

Label not defined: 31517
Includes_Shelf_Contents.xml: line 41

Label not defined: 31497
Includes_Shelf_Contents.xml: line 116

Label not defined: 31517
Includes_Shelf_Contents.xml: line 127

Label not defined: 31497
Includes_Shelf_Contents.xml: line 149

Label not defined: 31517
Includes_Shelf_Contents.xml: line 160

Label not defined: 31497
Includes_Shelf_Contents.xml: line 223

Label not defined: 31517
Includes_Shelf_Contents.xml: line 234

Label not defined: 31497
Includes_Shelf_Contents.xml: line 287

Label not defined: 31517
Includes_Shelf_Contents.xml: line 298

Label not defined: 32104
CustomVideoInfoLists.xml: line 33

Label not defined: 32105
CustomVideoInfoLists.xml: line 60

Label not defined: 32106
CustomVideoInfoLists.xml: line 97

Label not defined: 32099
CustomVideoInfoLists.xml: line 103

Label not defined: 32082
Custom_1128_TMDB_PersonInfo.xml: line 140

Label not defined: 32083
Custom_1128_TMDB_PersonInfo.xml: line 146

Label not defined: 32085
Custom_1128_TMDB_PersonInfo.xml: line 158

Label not defined: 32086
Custom_1128_TMDB_PersonInfo.xml: line 164

Label not defined: 32137
script-embuary-video.xml: line 367

Label not defined: 32138
script-embuary-video.xml: line 373

Label not defined: 32073
script-embuary-video.xml: line 408

Label not defined: 32074
script-embuary-video.xml: line 414

Label not defined: 32075
script-embuary-video.xml: line 420

Label not defined: 32097
script-embuary-video.xml: line 603

Label not defined: 32105
script-embuary-video.xml: line 691

Label not defined: 32131
script-embuary-video.xml: line 735

Label not defined: 32130
script-embuary-video.xml: line 823

Label not defined: 32157
Custom_1132_ColorBoxSettings.xml: line 39

Label not defined: 32158
Custom_1132_ColorBoxSettings.xml: line 82

Label not defined: 32159
Custom_1132_ColorBoxSettings.xml: line 118

Label not defined: 32160
Custom_1132_ColorBoxSettings.xml: line 127

Label not defined: 32161
Custom_1132_ColorBoxSettings.xml: line 136

Label not defined: 32162
Custom_1132_ColorBoxSettings.xml: line 197

Label not defined: 32163
Custom_1132_ColorBoxSettings.xml: line 226

Label not defined: 32164
Custom_1132_ColorBoxSettings.xml: line 226

Label not defined: 32120
DialogContextMenu.xml: line 43

Label not defined: 32121
DialogContextMenu.xml: line 61

Label not defined: 32082
script-embuary-person.xml: line 132

Label not defined: 32083
script-embuary-person.xml: line 138

Label not defined: 32085
script-embuary-person.xml: line 150

Label not defined: 32136
script-embuary-person.xml: line 156

Label not defined: 32134
script-embuary-person.xml: line 172

Label not defined: 32095
DialogButtonMenu.xml: line 130

Label not defined: 32073
DialogVideoInfo.xml: line 1008

Label not defined: 32074
DialogVideoInfo.xml: line 1014

Label not defined: 32075
DialogVideoInfo.xml: line 1020

Label not defined: 32077
DialogVideoInfo.xml: line 1030

Label not defined: 32078
DialogVideoInfo.xml: line 1036

Label not defined: 32079
DialogVideoInfo.xml: line 1042

Label not defined: 32080
DialogVideoInfo.xml: line 1048

Label not defined: 32081
DialogVideoInfo.xml: line 1057

Label not defined: 32154
DialogVideoInfo.xml: line 1111

Label not defined: 32127
DialogVideoInfo.xml: line 1150

Label not defined: 32135
DialogVideoInfo.xml: line 1162

Label not defined: 32123
SkinSettings.xml: line 86

Label not defined: 32107
SkinSettings.xml: line 90

Label not defined: 32156
SkinSettings.xml: line 435

Label not defined: 32156
SkinSettings.xml: line 444

Label not defined: 32157
SkinSettings.xml: line 453

Label not defined: 32104
SkinSettings.xml: line 458

Label not defined: 32128
SkinSettings.xml: line 463

Label not defined: 32155
SkinSettings.xml: line 468

bartolomesoriano commented:
@ronie , thanks! I am not sure what happened with the language file.  I have updated the English language files and removed the BOM character.  Please check now.  Thanks.

ronie commented:
thx!
